package.json

```diff
@@ -62,7 +62,7 @@
     "proxyquire": "^2.1.3",
     "rimraf": "^5.0.1",
     "ts-mocha": "^10.0.0",
-    "typescript": "^5.1.6"
+    "typescript": "~5.1.6"
```

Comment:
I wanted to confirm that the tests were failing already on master, so I had to "~lock" this to 5.1 so that the older balena-lint still works.
Will unlock it and bump it to 5.2 in the balena-lint bump PR
See: https://github.com/balena-io-modules/balena-compose/pull/35